src/uid2StorageManager.ts

```diff
@@ -0,0 +1,48 @@
+import { UID2CookieManager } from "./uid2CookieManager";
+import { Uid2Identity } from "./Uid2Identity";
+import { UID2LocalStorageManager } from "./uid2LocalStorageManager";
+import { Uid2Options } from "./Uid2Options";
+
+export class UID2StorageManager {
+  private _cookieManager: UID2CookieManager | undefined;
```

Comment:
I dropped you a DM about this - I think the coverage is showing as low on this file because of these `| undefined`, and these values will always be set once the constructor has run.